tests/functional/sync_v3_sign_up.js

```diff
@@ -91,9 +91,9 @@ define([
         .then(testIsBrowserNotified('fxaccounts:can_link_account'))
         .then(openVerificationLinkInDifferentBrowser(email, 0))
 
-        // In fx >= 57, about:accounts does not take over.
+        // In Fx >= 57, about:accounts does not take over.
         // Expect a screen transition.
-        .then(testElementExists(selectors.SIGNUP_COMPLETE.HEADER))
+        .then(testElementExists(selectors.CONNECT_ANOTHER_DEVICE.HEADER))
```

Comment:
Here's another - Here's one assertion to check that the user goes to CAD instead!